client/webpack/paths.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 module.exports = {
   root: path.resolve(__dirname, '../'),
   outputPath: path.resolve(__dirname, '../', 'build'),
-  entryPath: path.resolve(__dirname, '../', 'src/index.js'),
+  entryPath: path.resolve(__dirname, '../', 'src/thunks.js'),
```

Comment:
not sure about this change... why did you need to change entry point?

Reply:
My fault, i forgot to change it back:(